assets/src/scripts/components/median-water-level-table/index.js

```diff
@@ -0,0 +1,117 @@
+import { select } from 'd3-selection';
+import { createStructuredSelector } from 'reselect';
+
+import List from 'list.js';
+
+import { link } from '../../lib/d3-redux';
+import {
+    getSiteMedianWaterLevels,
+    getMedianWaterLevelStatus,
+    retrieveMedianWaterLevels
+} from '../../services/state';
+
+import { isTableRendered, renderTable } from './state';
+
+const COLUMN_HEADINGS = [
+    'Year',
+    'Month',
+    'Median'
+];
+
+/*
+ * Draws the waterLevels data into table
+ * @param {D3 node} table
+ * @param {Array of Object} waterLevels
+ * @return {Object} - Return the tbody element for the table
+ */
+const drawTableBody = function(table, waterLevels, tbody) {
+    tbody = tbody || table
+        .append('tbody')
+        .classed('list', true);
+    const samples = waterLevels.medians || [];
+    const valueNames = ['year', 'month', 'median'];
+    const item = valueNames.reduce(function(total, name) {
+        return `${total}<td class="${name}"></td>`;
+    }, '');
+    const options = {
+        valueNames: valueNames,
+        item: `<tr>${item}</tr>`,
+        page: 30,
+        pagination: true
+    };
+    new List('median-water-levels-div', options, samples);
+
+    return tbody;
+};
+
+/*
+ * Renders the median water level table
+ * @param  {Object} store               Redux store
+ * @param  {Object} node                DOM node to draw graph into
+ * @param  {Object} options.agencyCode  Agency of site to draw
+ * @param  {Object} options.siteId      ID of site to draw
+ * @param  {String} options.id          Unique ID for this component
+ */
+const drawTable = (store, node, agencyCode, siteId) => {
+    const component = select(node);
+    const table = component
+        .select('#median-water-levels-div')
+        .append('table')
+        .attr('id', 'median-water-levels-table')
+        .classed('usa-table', true);
```

Comment:
Here's a suggestion for a new places in this rendering code... A pattern we've been following is to indent after appending a new node, like this:

```js
        .append('table')
            .attr('id', 'median-water-levels-table')
            .classed('usa-table', true);
```

It tends to help with comprehending what node the function calls are applying to.

Reply:
ah, it looked like a mistake to me, I "fixed" it throughout the code as I discovered them.